lib/libyui_client/widgets/menubutton.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module LibyuiClient
+  module Widgets
+    # Class representing a menubutton in UI. It can be YMenuButton.
+    class Menubutton < Widgets::Base
+      # Sends action to click on one of the items of the menubutton in UI.
+      # @param item [String] value to select from items.
+      # @example Click button with id 'test'
+      #   app.menubutton(id: 'test_id', value: test_item).click
```

Comment:
As I understand from the code below, the click should be made with different syntax: `app.menubutton(id: 'test_id').click('test_item')`. Could you please update the documentation.

Reply:
good catch, fixed.